packages/compass-sidebar/src/components/sidebar.tsx

```diff
@@ -153,40 +179,44 @@ export function Sidebar({
       expanded={isExpanded}
       setExpanded={setIsExpanded}
       data-testid="navigation-sidebar"
+      className={sidebarStyles}
     >
       <>
-        <SidebarTitle
-          title={getConnectionTitle(connectionInfo)}
-          isFavorite={!!connectionInfo.favorite}
-          isExpanded={isExpanded}
-          onAction={onAction}
-        />
-        {connectionInfo.favorite && (
-          <FavoriteIndicator favorite={connectionInfo.favorite} />
+        {showConnectionInfo && (
+          <div className={connectionInfoContainerStyles}>
```

Comment:
Did some restyling here to fix the most obvious visual issues when using this in web environment with the connection info secion missing, functionally it's all the same as before and looks the same in electron